### ISSUE TITLE
The loss is nan

### ISSUE THREAD

**Issue by hkzhang91** [apple/ml-cvnets#2]
I'm trying to train the classificaion model in my machine using 8 RTX 2080ti. Limited the memory size of GPU, I set batchsize  for each GPU to 32. After training several iterations or several epoches, loss become NaN and the training is stopped. Could you tell me how to solve this problem?

**Comment by sacmehta** [comment]
You are using smaller batch size. So, you need to adjust the learning rate accordingly.

You may want to change the learning rate according to the learning rate scaling rule mentioned in below paper.

https://arxiv.org/pdf/1706.02677.pdf

**Comment by sacmehta** [comment]
No activity, so closing it
